buildrunner/docker/multiplatform_image_builder.py

```diff
@@ -558,13 +558,9 @@ def push(self) -> None:
         Pushes all built images to their tagged image counterparts.
         :raises TimeoutError: If the image fails to push within the specified timeout and retries
         """
-        # Parameters for timeout and retries
-        initial_timeout_seconds = 60
-        timeout_step_seconds = 60
-        timeout_max_seconds = 600
-        retries = 5
```

Comment:
These parameters are no longer need when this was refactored a while ago and `@timeout_decorator.timeout(PUSH_TIMEOUT)` was used.